pkg/command/utils/util_test.go

```diff
@@ -143,3 +143,16 @@ func TestGenerateJSONFile(t *testing.T) {
 		assert.ErrorContains(t, err, "failed to create json file open /tmp: is a directory")
 	})
 }
+
+func TestCheckOutputLocation(t *testing.T) {
+	t.Run("dir not exist", func(t *testing.T) {
+		dirName := "/tmpdbcd"
+		_, err := CheckOutputLocation(dirName)
+		assert.ErrorContains(t, err, "is not existed")
+	})
+	t.Run("pass a filename", func(t *testing.T) {
+		dirName := "./util_test.go"
+		_, err := CheckOutputLocation(dirName)
+		assert.ErrorContains(t, err, "is not directory")
+	})
```

Comment:
Could you add a test case for "error dir" such as an unreadable directory.

Reply:
Thank you, I have added a check for not writable, and also a testcase for it.

Reply:
Good job!  And leave tiny comments.